python/ray/scripts/scripts.py

```diff
@@ -229,6 +240,10 @@ def start(node_ip_address, redis_address, redis_port, num_redis_shards,
                         "    --resources='{\"CustomResource1\": 3, "
                         "\"CustomReseource2\": 2}'")
 
+    if include_java is None and java_worker_options is not None:
+        raise Exception("Should not specify `java-worker-options`"
```

Comment:
- `ValueError` is better for bad arguments.
- we can put this check in `RayParam._check_usage`